plugins/main/public/controllers/overview/components/last-alerts-stat/last-alerts-query.ts

```diff
@@ -27,6 +31,14 @@ export const getLastAlertsQuery = (
           },
         },
       },
+      {
+        range: {
+          'rule.level': {
+            gte: ruleLevelRange.minRuleLevel,
+            lte: ruleLevelRange.maxRuleLevel || 15,
```

Comment:
**suggestion**: the `15` value could be a constant instead.

Reply:
The query should be able to support only a minimum value of a range. I'll fix it.